less/components/currency-dropdown.less

```diff
@@ -3,6 +3,10 @@
   text-transform: uppercase;
   text-align: center;
   padding: 0 20px;
+  position: absolute;
+  width: 125px;
+  right: 5px;
+  top: 0;
```

Comment:
Oh wow, now that I see this, the currency dropdown is IN the h3, which it really shouldn't.

We should fix them, and put them side by side.